indigo_api/data_migrations/__init__.py

```diff
@@ -72,3 +72,25 @@ def strip_whitespace(self, xml):
 
         return xml
 
+
+class UpdateAKNNamespace(AKNMigration):
+    """ Update all instances of:
+    <akomaNtoso xmlns="http://www.akomantoso.org/2.0" xsi:schemaLocation="http://www.akomantoso.org/2.0 akomantoso20.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    to
+    <akomaNtoso xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0" xsi:schemaLocation="http://docs.oasis-open.org/legaldocml/ns/akn/3.0 ../akomantoso30.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    """
+    # TODO: Sanity check use and value of xsi:schemaLocation
+    def update_namespace(self, xml):
+        from cobalt import Act
```

Comment:
You could probably just do this with a simple regex replacement on the raw XML. No need to parse it with Cobalt.

Reply:
I suggest three steps:

1. remove ` xsi:schemaLocation="http://www.akomantoso.org/2.0 akomantoso20.xsd"`
2. remove ` xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"`
3. change `xmlns="..."`

(note the leading space on the first two)